tensorflow/tools/ci_build/ci_parameterized_build.sh

```diff
@@ -83,8 +83,8 @@
 #                      support for Google Cloud Platform (GCP), which is
 #                      enabled by default.
 #   TF_BUILD_OPTIONS:
-#                     (FASTBUILD | OPT | OPTDBG | MAVX | MAVX2 | MAVX_DBG |
-#                      MAVX2_FMA_DBG)
+#                     (FASTBUILD | OPT | OPTDBG | MAVX | MAVX2 | MAVX2_FMA |
```

Comment:
Let's instead change MAVX2 build to also enable FMA, they come together anyway.
To make the naming more understandable you can remove MAVX2